core/src/com/unciv/logic/map/UnitMovementAlgorithms.kt

```diff
@@ -8,6 +8,8 @@ class UnitMovementAlgorithms(val unit:MapUnit) {
 
     // This function is called ALL THE TIME and should be as time-optimal as possible!
     fun getMovementCostBetweenAdjacentTiles(from: TileInfo, to: TileInfo, civInfo: CivilizationInfo): Float {
+        if (unit.allTilesCosts1)
```

Comment:
I'm pretty sure that helicopters also get the +1 movement penalty from the great wall

Reply:
-Reverted mountains not giving sight bonus. I think I was misled by the comment in the units json file saying that mountains don't give a vision bonus. Seems that was incorrect? I checked in Civ 5 and helicopters on mountains can see over hills.
-Helicopters do not get a movement penalty from great wall. Tested in Civ 5 using map editor and loading the scenario - I tested with a knight which did get a movement penalty and I tested with a helicopter which didn't get a movement penalty.